requirements.txt

```diff
@@ -1,5 +1,5 @@
-bokeh>=1.0.2
+bokeh>=1.0.2,<1.3.0
 featuretools>=0.6.0
```

Comment:
can we bump this to 0.16

Reply:
We can, but we don't need to as this particular notebook will still work with older versions of Featuretools. Probably would be best to bump though, in light of the comment I added to the notebook regarding naming of cutoff time columns.

Reply:
Bumped